python/sbp/client/framer.py

```diff
@@ -119,7 +120,7 @@ def _receive(self):
         crc = self._readall(2)
         crc, = struct.unpack("<H", crc)
         if crc != msg_crc:
-            print "crc mismatch: 0x%04X 0x%04X" % (msg_crc, crc)
+            print("crc mismatch: 0x%04X 0x%04X" % (msg_crc, crc))
```

Comment:
It would be nice if there was a verbosity API that let CRC mismatch print be selectively printable like the "Host Side Unhandled byte"

Reply:
ok, putting the printout under the same _verbose flag which is set in Framer constructor. Console and some cli tools already set this flag based on the command line -v option, and I can add it to settings and bootload_v3 which currently miss it.